src/state/stake/hooks.ts

```diff
@@ -545,13 +544,12 @@ export function useSyrupInfo(
           const getHypotheticalRewardRate = (
             stakedAmount: TokenAmount,
             totalStakedAmount: TokenAmount,
-            totalRewardRate: TokenAmount,
           ): TokenAmount => {
             return new TokenAmount(
               token,
               JSBI.greaterThan(totalStakedAmount.raw, JSBI.BigInt(0))
                 ? JSBI.divide(
-                    JSBI.multiply(totalRewardRate.raw, stakedAmount.raw),
+                    JSBI.multiply(totalRewardRate01.raw, stakedAmount.raw),
```

Comment:
What is rewardsRate01 this name inls confusing please provide a more clear name on what that variable represents.

Reply:
changed `totalRewardRate01` to `rewardRate`